test/NotEmptyTest.php

```diff
@@ -27,6 +27,33 @@ public function setUp()
         $this->validator = new NotEmpty();
     }
 
+    public function testConstructorWithTypeArray()
+    {
+        $validator = new NotEmpty([
+            'php',
```

Comment:
Not sure what you mean here, the integer and zero bitmasks don't overlap, so they don't cause the issue that I'm fixing.